toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -18,7 +43,7 @@ common_options="-c $repo_file_path --releasever $mariner_version"
 
 DNF_COMMAND=dnf
 # Cache RPM metadata
->/dev/null dnf $common_options -y makecache
+>/dev/null $DNF_COMMAND $common_options -y makecache
```

Comment:
Is `DNF_COMMAND` needed, if it never changes and doesn't add any extra arguments to `dnf`?

Reply:
The DNF_COMMAND variable is mainly there for easy refactoring in the future (if/when we change over to dnf5 or other lightweight manager in the future).

If there is no true objection, I would like to leave it in place but changed the var to all lower case for consistency in naming.